xapian-letor/api/featuremanager_internal.cc

```diff
@@ -39,156 +39,156 @@ using namespace std;
 
 
 std::string
-FeatureManager::Internal::getdid(const Document &doc) 
+FeatureManager::Internal::getdid(const Document &doc)
 {
-    string id="";
-    string data = doc.get_data();
-    string temp_id = data.substr(data.find("url=", 0), (data.find("sample=", 0) - data.find("url=", 0)));
-    id = temp_id.substr(temp_id.rfind('/') + 1, (temp_id.rfind('.') - temp_id.rfind('/') - 1));  //to parse the actual document name associated with the         documents if any
+	string id="";
+	string data = doc.get_data();
+	string temp_id = data.substr(data.find("url=", 0), (data.find("sample=", 0) - data.find("url=", 0)));
+	id = temp_id.substr(temp_id.rfind('/') + 1, (temp_id.rfind('.') - temp_id.rfind('/') - 1));  // to parse the actual document name associated with the         documents if any
 
-    return id;
+	return id;
 }
 
 int
-FeatureManager::Internal::getlabel(map<string, map<string, int> > qrel2, const Document &doc, std::string & qid) 
+FeatureManager::Internal::getlabel(map<string, map<string, int> > qrel2, const Document &doc, std::string & qid)
 {
-    int label = -1;
-    string id = getdid(doc);    
+	int label = -1;
+	string id = getdid(doc);
 
-    map<string, map<string, int> >::iterator outerit;
-    map<string, int>::iterator innerit;
+	map<string, map<string, int> >::iterator outerit;
+	map<string, int>::iterator innerit;
 
-    outerit = qrel2.find(qid);
-    if (outerit != qrel2.end()) {
+	outerit = qrel2.find(qid);
+	if (outerit != qrel2.end()) {
 	innerit = outerit->second.find(id);
 	if (innerit != outerit->second.end()) {
-        label = innerit->second;
+		label = innerit->second;
 	}
-    }
-    return label;
+	}
+	return label;
 }
 
 Xapian::RankList
-FeatureManager::Internal::create_rank_list(const Xapian::MSet & mset, std::string & qid) 
+FeatureManager::Internal::create_rank_list(const Xapian::MSet & mset, std::string & qid)
 {
-    Xapian::RankList rl;
-
-    for (Xapian::MSetIterator i = mset.begin(); i != mset.end(); ++i) {
-        
-        Xapian::Document doc = i.get_document();
-        
-        // Here a weight vector can be created in future for different weights of the document 
-        // like BM25, LM etc. 
-        double weight = i.get_weight();
-
-        map<int,double> fVals = transform(doc, weight);
-        string did = getdid(doc);
-        int label = getlabel(qrel, doc, qid);
-
-        if(label!=-1) {
-            Xapian::FeatureVector fv = create_feature_vector(fVals, label, did);
-            rl.set_qid(qid);
-            rl.add_feature_vector(fv);
-        }
-    }
-    rl.normalise();
-    return rl;
+	Xapian::RankList rl;
```

Comment:
This seems to have an indent of 8 (which is expanding out to a tab) rather than 4?